drainer/sync/plugin_demo.go

```diff
@@ -0,0 +1,56 @@
+package sync
+
+import (
+	"fmt"
+
+	"github.com/pingcap/log"
+	"github.com/pingcap/tidb-binlog/drainer/loopbacksync"
+	"github.com/pingcap/tidb-binlog/drainer/relay"
+	"github.com/pingcap/tidb-binlog/drainer/translator"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/zap"
+)
+
+//SyncerDemo is a syncer demo
+type SyncerDemo struct {
+	*baseSyncer
+}
+
+//Sync is the method that interface must implement
+func (sd *SyncerDemo) Sync(item *Item) error {
+	//demo
+	log.Info("item", zap.String("%s", fmt.Sprintf("%v", item)))
+	sd.success <- item
+	return nil
+}
+
+//Close is the method that interface must implement
+func (sd *SyncerDemo) Close() error {
+	return nil
+}
+
+//SetSafeMode is the method that interface must implement
+func (sd *SyncerDemo) SetSafeMode(mode bool) bool {
+	return false
+}
+
+//NewSyncerDemo is a syncer demo
+func NewSyncerDemo(
```

Comment:
how about adding some comments for how to use these arguments?  For example, I don't know what the `file` used for.

Reply:
Thanks, I had writen a READ.md. PTAL @WangXiangUSTC